connectors/email/src/main/java/io/camunda/connector/email/client/jakarta/utils/JakartaUtils.java

```diff
@@ -310,9 +310,16 @@ private void processBodyPart(
     }
   }
 
-  private String stripMessageId(String messageId) {
-    if (messageId == null) return null;
-    return messageId.trim().replaceAll("[<>]", "");
+  private String getMessageId(Message message) {
+    try {
+      String[] messageIds = message.getHeader("Message-ID");
```

Comment:
This can throw a Nullpointer exception. so maybe this would be more robust: `return (messageIds != null && messageIds.length > 0) ? messageIds[0] : null;`

Reply:
Good point! @mathias-vandaele was this fixed?